### PR TITLE
Lps 88439 another

### PR DESCRIPTION
**Pull request by arthurchan35**
Hey Carlos, let me if this is a proper way to handle the JSPs. If so, I will do some cleanup later.

### CODE REVIEW

File: modules/dxp/apps/saml/saml-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -1,6 +1,7 @@
 a-further-x-entries-of-other-types-were-ignored=A further {0} entries of other types were ignored.
```
Comment (liferay-appsec) [inline modules/dxp/apps/saml/saml-web/src/main/resources/content/Language.properties:1]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:test:saml

**Comment by stian-sigvartsen** [comment]
Hi @arthurchan35 . These changes create dependencies on the next release of portal and the frontend themes. Did you dismiss ways to avoid this already?

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:test:saml

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:stop

**Comment by arthurchan35** [comment]
ci:reevaluate:1087401_7093

**Comment by arthurchan35** [comment]
close as the ci is broken

**Comment by arthurchan35** [comment]
ci:stop

**Pull request closed without merging**